### DISCUSSION TITLE
A problem about the infty boundary condition

### DISCUSSION THREAD

**Discussion by possible7** [lululxvi/deepxde#1082] [Q&A]
Hi, Dr. Lu Lu

Thank you once again for this great library.

The documentation explains very clearly how to impose various boundary conditions. However, suppose I'm solving a one-dimensional ODE u(x) and I want to impose $\lim _{x \rightarrow+\infty} u(x)=1$. How can I do that? Is this possible through deepxde?

**Comment by lululxvi** [comment]
It is not straightforward. You can borrow the ideas in traditional numerical methods for handling infinity domain.
